src/EntityFramework6.Npgsql/EntityFramework6.Npgsql.nuspec

```diff
@@ -16,7 +16,7 @@
     <language>en-US</language>
     <tags>npgsql postgresql postgres data database entity framework ef orm</tags>
     <dependencies>
-      <dependency id="Npgsql" version="3.1.0" />
+      <dependency id="npgsql" version="3.1.0" />
       <dependency id="EntityFramework" version="6.0.0" />
```

Comment:
Last I checked, the nuget versioning strategy is to always take the lowest possible version which satisfies all requirements. Since EF6.Npgsql have been decoupled from Npgsql releases (in 3.1) it's up to users to specify which versions they want for which component. This removes the need to publish a new EF6.Npgsql package even if nothing's changed, just because a new Npgsql has been released, and also gives more control to users.

So the idea is that as long as EF6.Npgsql is compatible with any Npgsql 3.1.0, it specifies 3.1.0 as its minimum dependency. If you want a higher version of Npgsql (which you should), you need to specify that in your own project.json.